ctms/schemas/__init__.py

```diff
@@ -5,6 +5,8 @@
     ContactPatchSchema,
     ContactPutSchema,
     ContactSchema,
+    CTMSBulkResponse,
+    CTMSOptionalResponse,
```

Comment:
I saw @jwhitlock add to the init for the PATCH, didn't realize this __init__ was in use, but added the two responses here. A CTMSBulkResponse has in it a List[CTMSOptionalResponse] which allows for ContactSchema or CTMSSingleResponse schemas to not have all parts of a contact, including amo or other by-table data